### ISSUE TITLE
Female Undead are missing their jaws.

### ISSUE THREAD

**Issue by benjymansy123** [benjymansy123/mod-worgoblin#21]
This is the first (Probably of a few) bugs related to the HD patch for future reference. This won't be fixed anytime soon, but I'm reporting it so I don't forget later on.

**Comment by yuanf225** [comment]
I'm currently using this most user-friendly 335HD client, https://discord.gg/Gyb9Z252vt, I recompiled AC and I'm going to try this module again, if you have any questions, I'll try to leave a message here
